operations/iuf/workflows/management_rollout.md

```diff
@@ -496,7 +497,19 @@ Once this step has completed:
 Return to the procedure that was being followed for `management-nodes-rollout` to complete the next step, either [Management-nodes-rollout with CSM upgrade](#31-management-nodes-rollout-with-csm-upgrade) or
 [Management-nodes-rollout without CSM upgrade](#32-management-nodes-rollout-without-csm-upgrade).
 
-## 4. Update management host Slingshot NIC firmware
+## 4. Restart `goss-servers` on all NCNs
+
+The `goss-servers` service needs to be restarted on all NCNs. This ensures the correct tests are run on each NCN. This is necessary due to a timing issue in the NCN's cloud-init which has been fixed in CSM 1.6.1.
+
+(`ncn-m001#`) Restart `goss-servers`.
+
+  ```bash
+  ncn_nodes=$(grep -oP "(ncn-s\w+|ncn-m\w+|ncn-w\w+)" /etc/hosts | sort -u | tr -t '\n' ',')
+  ncn_nodes=${ncn_nodes%,}
+  pdsh -S -b -w $ncn_nodes 'systemctl restart goss-servers'
+  ```
```

Comment:
I don't think this should be indented.

````suggestion
```bash
ncn_nodes=$(grep -oP "(ncn-s\w+|ncn-m\w+|ncn-w\w+)" /etc/hosts | sort -u | tr -t '\n' ',')
ncn_nodes=${ncn_nodes%,}
pdsh -S -b -w $ncn_nodes 'systemctl restart goss-servers'
```
````

Reply:
Fixed